.golangci.yml

```diff
@@ -9,4 +9,4 @@ linters:
     - goimports
 linters-settings:
   goimports:
-    local-prefixes: github.com/snapp-incubator/s3-operator
+    local-prefixes: github.com/snapp-incubator/ceph-s3-operator
```

Comment:
Are you sure whether this will change :-?
If yes, kindly please resolve this.

Reply:
It changes. The old addresses will redirect to the new address.

https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository